src/client.ts

```diff
@@ -43,6 +52,27 @@ export interface ClickHouseClientConfigOptions {
   }
   tls?: BasicTLSOptions | MutualTLSOptions
   session_id?: string
+  /** HTTP Keep-Alive related settings */
+  keep_alive?: {
+    /** Enable or disable HTTP Keep-Alive mechanism. Default: true */
+    enabled?: boolean
+    /** Sockets housekeeping settings */
+    sockets?: {
+      /** How long to keep a particular open socket alive (in milliseconds).
+       * Should be less than the server setting
+       * (see `keep_alive_timeout` in server's `config.xml`).
+       * Default value: 2500
+       * (based on the default ClickHouse server setting, which is 3000) */
+      socket_ttl: number
```

Comment:
Why not timeout to follow the standard term? To avoid mixing it up with the built-in socket timeout?

Reply:
Yes, the `timeout` term is used a bit differently - that one will happen when there is no _data_ sent to the socket after X milliseconds.